packages/devextreme/js/__internal/grids/grid_core/columns_controller/m_columns_controller.ts

```diff
@@ -501,6 +502,17 @@ export class ColumnsController extends modules.Controller {
     return 0;
   }
 
+  public getStickyColumns(): any[] {
+    const visibleColumns = this.getVisibleColumns(null, true);
+    const isColumnFixing = this._isColumnFixing();
```

Comment:
The result is not related to this._isColumnFixing() method. It seems it should return anyway:
`visibleColumns.filter((column) => column.fixed);`